includes/ucf-faq-posttype.php

```diff
@@ -83,6 +83,10 @@ public static function args( $singular, $plural ) {
 				'capability_type'       => 'post',
 			);
 
+			if( UCF_FAQ_Config::get_option_or_default( UCF_FAQ_Config::$option_prefix . 'disable_faq_archive' ) ) {
+				$args['has_archive'] = false;
```

Comment:
We may want to consider adding a hook somewhere that will force a permalink flush when this archive setting is changed, so it doesn't have to be done manually.

Could probably use `update_option_{$option}`: https://developer.wordpress.org/reference/hooks/update_option_option/